src/fragments/lib/auth/js/advanced.mdx

```diff
@@ -240,42 +240,6 @@ const SignInWithGoogle = () => {
 }
 ```
 
-### Using Amplify UI components
```

Comment:
This section is removed because Authenticator now does social sign in through User Pools, and not through identity pool that this docs describes. So I moved this UI docs reference out of this section and to social provider docs (see below).